addons/l10n_mx/models/account.py

```diff
@@ -48,7 +48,8 @@ def mx_search_tags(self, code):
     def _onchange_code(self):
         if self.company_id.country_id.id == self.env.ref('base.mx').id and self.code:
             tags = self.mx_search_tags(self.code)
-            self.tag_ids = tags
+            self.tag_ids = self.tag_ids.filtered(
```

Comment:
@moylop260 Thanks for the explanaition, i just upload the change.

About the last commet, if that logic is used in mx_search_tags method, How this method will be affected? https://github.com/Vauxoo/odoo/blob/10.0/addons/l10n_mx/models/account.py#L13. Is it expecting the return only the tags for mexican localization?